features/largeVideo/components/LargeVideo.js

```diff
@@ -19,10 +22,24 @@ class LargeVideo extends Component {
      * @returns {ReactElement}
      */
     render() {
+        const avatar = this.props.avatar;
+        const videoTrack = this.props.videoTrack;
+        const renderAvatar
+            = typeof avatar !== 'undefined'
+                && avatar !== ''
+                && (!videoTrack
+                    || (videoTrack
```

Comment:
If `!videoTrack` is false, then `videoTrack` is true. Consequently, there's no point in the seconds check of `videoTrack`.